config.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+def loadEnv(key):
+    value = os.environ.get(key)
+    if not value:
+        raise ValueError(f"No value set for {key}")
+    return value
+
+TRELLO_KEY = loadEnv('TRELLO_KEY')
```

Comment:
(Minor) You could move these to class attributes to simplify some of the import statements